Problem1_usingTwoBinarySearch.java

```diff
@@ -0,0 +1,86 @@
+//PROBLEM - Find First and Last Position of Element in Sorted Array
+/** 3 Pointer Approcach: 
+ * Fact utilized : If we can get half items to compute at each progressive iterarion  
+ * the problem can be resolved in Log N time.
+ * Here we have rotated array, however at any given iteration we will always have 
+ * half array sorted - either left or rigth.
+ * First find the First occurance of Number by using Binary Search
+ * From that position found First Occurance and Last Occurance Using Find First and Find Last.
+ */ 
+//
+//time Complexity :
+// LOG(N)
+
+// Space Complexity :
+//  LOG(N)
+
+// Did this code successfully run on Leetcode : YES.
+// Any problem you faced while coding this : Faced some challage while considering the corner cases => 0th and Nth index.
+
+class Solution {
+    public int[] searchRange(int[] nums, int target) {
+
+        int index = BinarySearch(nums, target, 0, nums.length - 1); // Find first occurance;
```

Comment:
What was the benefit of performing an extra Binary Search and finding the first occurrence twice?
If you plan to reply, please do so on Slack.